docs/getting-started/configuration.md

```diff
@@ -0,0 +1,39 @@
+## Disable Forwarding Config
```

Comment:
lgtm! can you please change this also here : https://github.com/ovn-org/ovn-kubernetes/blob/47fcdd2607be4e4b9cc198a0d6fca385bf4030ed/mkdocs.yml#L89 so that it appears on our website?
use https://ovn-kubernetes.io/developer-guide/documentation/#how-to-test-your-documentation-changes to test your changes

Reply:
Done